### ISSUE TITLE
Superimposition textures to object

### ISSUE THREAD

**Issue by Disinterpreter** [multitheftauto/mtasa-blue#1709]
**Is your feature request related to a problem? Please describe.**
We have the issue https://github.com/multitheftauto/amx/issues/23 and I think we should implement this feature in MTA.

**Describe the solution you'd like**
In nutshell, we haven't functions that can attach the texture or text to object.

I'd like to make two functions in clientside like
```lua
dxSetTextureObject(objectid, materialindex, modelid, txdname, texturename, materialcolor)

dxSetTextureObjectText(objectid, text, materialindex = 0, materialsize = OBJECT_MATERIAL_SIZE_256x128, fontface = "Arial", fontsize = 24, bold = 1, fontcolor = 0xFFFFFFFF, backcolor = 0, textalignment = 0)
```

**Describe alternatives you've considered**
\-

**Additional context**
https://wiki.open.mp/docs/scripting/functions/SetObjectMaterial
https://wiki.open.mp/docs/scripting/functions/SetObjectMaterialText
https://forum.gamerxserver.com/forum/samp-server-188-165-219-63-8800/questions-and-answers/tutorials/2907-object-material-tutorial-spoma

My function's names and arguments can be bad and inconvenient.
You can call it different name.

**Comment by Pirulax** [comment]
`engineGetModelTextures` + a texture replacement shader (with color overlay if you wish)

**Comment by AlexTMjugador** [comment]
I agree a shader could be the way to go, and I think it is easy to both change the texture color and the texture itself with it. However, how does `SetObjectMaterialText` work? Does it add a simple text texture on top of the already existing texture, which can then be drawn on existing geometry via shaders? How does it compute where the text goes? The links you provided only show how texture replacement works.
